File: src/main/java/me/kuku/yuq/service/RecallService.java
package me.kuku.yuq.service;

import com.IceCreamQAQ.Yu.annotation.AutoBind;
import me.kuku.yuq.entity.RecallEntity;

import java.util.List;

@AutoBind
public interface RecallService {
    List<RecallEntity> findByGroupAndQQ(Long group, Long qq);
    void save(RecallEntity recallEntity);
}

File: src/main/kotlin/me/kuku/yuq/service/RecallService.kt
package me.kuku.yuq.service

import com.IceCreamQAQ.Yu.annotation.AutoBind
import me.kuku.yuq.entity.RecallEntity

@AutoBind
interface RecallService {
    fun findByGroupAndQQ(group: Long, qq: Long): List<RecallEntity>
    fun save(recallEntity: RecallEntity)
}